plugins/vocabbuilder.koplugin/main.lua

```diff
@@ -0,0 +1,1180 @@
+
+    local subtitle_prefix = TextWidget:new{
+        text = BD.mirroredUILayout() and self:getTimeSinceDue() .. _("From ") or self:getTimeSinceDue() .. _("From ") ,
+        face = subtitle_face,
+        fgcolor = subtitle_color
+    }
+
+    self[1] = FrameContainer:new{
+        padding = 0,
+        bordersize = 0,
+        HorizontalGroup:new{
+            dimen = Geom:new{
+                w = self.width,
+                h = self.height,
+            },
+            HorizontalGroup:new{
+                dimen = Geom:new{
+                    w = self.width - right_side_width,
+                    h = self.height,
+                },
+                VerticalGroup:new{
+                    dimen = Geom:new{w = point_widget_width, h = self.height},
+                    self.point_v_spacer,
+                    point_widget,
+                    VerticalSpan:new { width = self.height - point_widget_height - self.point_v_spacer.width}
+                },
+                VerticalGroup:new{
+                    dimen = Geom:new{
+                        w = text_max_width,
+                        h = self.height,
+                    },
+                    self.v_spacer,
+                    LeftContainer:new{
+                        dimen = Geom:new{w = text_max_width, h = word_height},
+                        word_widget
+                    },
+                    LeftContainer:new{
+                        dimen = Geom:new{w = text_max_width, h = self.height - word_height - self.v_spacer.width*2.2},
+                        HorizontalGroup:new{
+                            subtitle_prefix,
+                            TextWidget:new{
+                                text = self.item.book_title,
+                                face = subtitle_italic_face,
+                                max_width = math.ceil(math.max(5,text_max_width - subtitle_prefix:getSize().w - Size.padding.fullscreen)),
+                                fgcolor = subtitle_color
+                            }
+                        }
+                    },
+                    self.v_spacer
+                }
+
+            },
+            RightContainer:new{
+                dimen = Geom:new{ w = right_side_width+Size.padding.default, h = self.height},
+                right_widget
+            }
+        },
+    }
+end
+
+function VocabItemWidget:getTimeSinceDue()
+    if self.item.review_count >= 8 then return "" end
+
+    local elapsed = os.time() - self.item.due_time
+    local abs = math.abs(elapsed)
+    local readable_time
+
+    local rounding = elapsed > 0 and math.floor or math.ceil
+    if abs < 60 then
+        readable_time = abs .. "s"
+    elseif abs < 3600 then
+        readable_time = string.format("%dm", rounding(abs/60))
+    elseif abs < 3600 * 24 then
+        readable_time = string.format("%dh", rounding(abs/3600))
+    else
+        readable_time = string.format("%dd", rounding(abs/3600/24))
+    end
```

Comment:
Pinging @zwim because I don't recall the exact name for the context otoh ("Time"?) but these should already be available in translation.